pages/mesosphere/dcos/services/kafka/2.8.1-2.3.1/limitations/index.md

```diff
@@ -14,7 +14,7 @@ render: mustache
 
 ## Log Retention Bytes
 
-The `disk` configuration value is denominated in MB. We recommend you set the configuration value `log_retention_bytes` to a value smaller than the indicated `disk` configuration. See the [Configuration](/mesosphere/dcos/services/kafka/2.4.0-1.1.1/configuration/) section for instructions on customizing these values.
+The `disk` configuration value is denominated in MB. It is recommended that you set the configuration value `log_retention_bytes` to a value smaller than the indicated `disk` configuration. See the [Configuration](/mesosphere/dcos/services/kafka/2.5.0-2.1.0/configuration/) section for instructions on customizing these values.
```

Comment:
Shouldn't this version match the package version? Right now it points to `2.5.0-2.1.0` but if it should be latest then it would be better to change to `/kafka/latest/configuration`